packages/core/plugins/core-plugins-server-internal/src/discovery/plugin_manifest_from_plugin_package.test.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { PluginPackageManifest } from '@kbn/repo-packages';
+import { PluginType } from '@kbn/core-base-common';
+import { pluginManifestFromPluginPackage } from './plugin_manifest_from_plugin_package';
+
+const kibanaVersion = `1.${Math.round(10 * Math.random())}.1`;
```

Comment:
NIT: `rand` in tests is bad, m'kay?

Reply:
lol, fine, want me to just put a random version here?

Reply:
Yeah, given we had surprises in the past when using the **current** version, having an hardcoded one seems the best compromise to me